apps/backend/src/frameworks/opencage.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { GeolocationService } from 'src/geolocation/geolocation.service';
```

Comment:
i think it might break our test.

```suggestion
import { GeolocationService } from '../core/abstracts/geolocation-service.abstract';
```